src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/HandlerUtils.java

```diff
@@ -51,6 +51,10 @@ public static boolean isInternalCase(AsylumCase asylumCase) {
         return (asylumCase.read(IS_ADMIN, YesOrNo.class)).orElse(YesOrNo.NO) == YesOrNo.YES;
     }
 
```

Comment:
Please add some comments here for future reference (Mainly to say in scope of EJP)